Source/WTF/wtf/PtrTag.h

```diff
@@ -51,6 +51,7 @@ enum class PACKeyType {
 
 enum PtrTag : uintptr_t {
     NoPtrTag,
+    OpaquePtrTag = NoPtrTag,
```

Comment:
I think we should put a comment here like:
// OpaquePtrTag is used for places where there is a tag but we intentionally have thrown away the diversifier. 
// For example, we register our WASM exception handler trampoline with a random key, pass the key to the OS, then throw the key away.
// So we have a tag but we can't verify it at runtime.

I think you also have to make this different from `NoPtrTag` because we assert in `CodePtr` that there's no tag on a `NoPtrTag` code.

Reply:
I turned it into a `void *` because I think realistically this abstraction doesn't really fit super well. I think `void *` is sufficient for the purposes